src/renderer/families/solana/shared/fields/ValidatorsField.js

```diff
@@ -27,14 +24,16 @@ type Props = {
   onChangeValidator: (v: ValidatorAppValidator) => void,
 };
 
-const ValidatorField = ({ t, account, onChangeValidator, chosenVoteAccAddr, status }: Props) => {
+const MaybeValidatorField = ({ status }: Props) => {
   if (!status) return null;
 
-  invariant(account && account.solanaResources, "solana account and resources required");
+  return ValidatorField;
+};
```

Comment:
This is here to replace the `if (!status) return null;` cause conditional hooks are not allowed by React.